docs/game/unreal/v4.2.7.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Unreal Engine v4
+id: setting-up-v4
+---
+# Setting up - v4
+
+Let's get started!
+
+# MacOS 
+
+The following instructions are based on Monterey v12.3.1.
+
+## Get Ready
+
+* Ensure you have the latest version of xcode.
+* Download the Epic Games Launcher to access the Unreal Engine. [**Download**](https://www.unrealengine.com/en-US/download?sessionInvalidated=true)
+* Install Unreal Engine v5. [**Download**](https://www.unrealengine.com/en-US/download?sessionInvalidated=true)
+
+
+## STEP ONE - Add v4 Unreal Engine
+
+1. Open **Epic Games Launcher** and view all versions of the **Unreal Engine** you have. 
+
+2. Click **Library** and the **+** button to add the **v4** version
+
+    <img src={require('/img/unreal-v5.png').default} alt="blank" width="600" />
+
+3. You can now click to install **v4**. (This takes some time)
+
+4. Launch **v4** (Be patient, this can take a little while)
+
+   :::note
+
+   If a pop up appears:
+   "Do you want the app **UnrealEditor.app** to accept incoming network connections"
+   Select **Allow**
+
+   <img src={require('/img/game-mac-1.png').default} alt="blank" width="200" />
+
+   :::
+
+## STEP TWO - Create a new project
+
+1. Inside the **Unreal Project Browser**, select **Games** and click **Next**
+
+   <img src={require('/img/v4-games-2.png').default} alt="blank" width="500" />
+
+2. On the **Template** screen, select **Blank** and click **Next**
+
+    :::note
+
+    If you can't see any templates, you should reinstall v4
+
+    :::
+
+    <img src={require('/img/game-template.png').default} alt="blank" width="500" />
+
+3. In **Project Settings**
+* From the **Blueprint** and **C++** options, select **C++**. 
+* From the **Starter Content** and **No Starter Content** options, select **No Starter Content**
```

Comment:
. missing 

you're using full sentences in your list. they usually do need a .